server/api/controllers/webhook/controller.ts

```diff
@@ -2,46 +2,25 @@ import axios from 'axios';
 import moment from 'moment';
 import { Request, Response } from 'express';
 import l from '../../../common/logger';
+import allData from './getData';
```

Comment:
`allData` is a bad variable name. Please use a better name. It should also match the filename it is importing from.

Reply:
For renaming allData keep the following things in mind.
- What is allData doing?
- What is the name given to the keys that we get from the database?